C#/Botkit/Microsoft.BotKit.Adapters.Slack/SlackBotWorker.cs

```diff
@@ -256,7 +247,11 @@ public void DialogError(AdapterError[] errors)
         /// <param name="dialogObj">A dialog, as created using [SlackDialog](#SlackDialog) or [authored to this spec](https://api.slack.com/dialogs).</param>
         public async Task<object> ReplyWithDialog(object source, object dialogObj)
```

Comment:
This could be changed from `public async Task<object>` to `public async Task<DialogOpenResponse>`

Reply:
Feedback applied. Thanks!